app/containers/HomePage.js

```diff
@@ -1,10 +1,6 @@
-import React, { Component } from 'react';
+import React from 'react';
 import Home from '../components/Home';
 
-export default class HomePage extends Component {
-  render() {
-    return (
-      <Home />
-    );
-  }
-}
+export default () => (
```

Comment:
Looks like one of the tests times out without the parentheses in Node 5. I'll try with the parentheses but inline.